SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -432,6 +432,14 @@ FRepChangeState USpatialActorChannel::CreateInitialRepChangeState(TWeakObjectPtr
 	for (uint16 CmdIdx = 0; CmdIdx < CmdCount; ++CmdIdx)
 	{
 		const auto& Cmd = Replicator.RepLayout->Cmds[CmdIdx];
+		const auto& Parent = Replicator.RepState->GetSendingRepState()->RepChangedPropertyTracker->Parents[Cmd.ParentIndex];
+
+		const bool RepActorMovement = Cmd.Type == ERepLayoutCmdType::RepMovement && Actor->GetReplicatedMovement().bRepPhysics;
```

Comment:
Really not sure about this, but apparently Active can be zero and bRepPhysics true, the expectation is that we replicate in that case. Maybe there is a better way of doing this?

Reply:
this sounds suspicious, I would try to see where the native code for handling this is

Reply:
I 'think' they subvert the entire replication system here `AActor::OnRep_ReplicatedMovement`

Reply:
Worth also putting a comment in saying what this is as it's hard to be hard for whoever sees this next to pick up

Reply:
"as it's hard to be hard" ? Mildly hilarious but also I don't understand what you mean here.

Reply:
bound to be hard is my guess

Reply:
Sorry I meant going to be hard.